relay_test.go

```diff
@@ -1046,11 +1046,17 @@ func TestRelayRaceTimerCausesStuckConnectionOnClose(t *testing.T) {
 }
 
 func TestRelayRaceCompletionAndTimeout(t *testing.T) {
-	const numCalls = 10
+	const numCalls = 100
 
 	opts := testutils.NewOpts().
 		AddLogFilter("simpleHandler OnError.", numCalls).
+		// Trigger deletion on timeout, see https://github.com/uber/tchannel-go/issues/808.
+		SetRelayMaxTombs(numCalls/2).
+		// Hitting max tombs will cause the following logs:
+		AddLogFilter("Too many tombstones, deleting relay item immediately.", numCalls).
+		AddLogFilter("Received a frame without a RelayItem.", numCalls).
 		SetRelayOnly()
+	opts.RelayMaxTombs = numCalls / 2
```

Comment:
Do we need both this and the one on L1054?

Reply:
Missed this once I added the option to the testutils, thanks